config/config.yaml

```diff
@@ -33,20 +33,20 @@ default:
                 path: ./mib
             datasink:
                 outgoing:
-                    path: ../bliss/cfdp/datasink/outgoing
+                    path: ../ait/cfdp/datasink/outgoing
```

Comment:
Should we change these to `../ait/dsn/cfdp/datasink...`?

Reply:
Good find. I didn't think about the directory updates there. I'll update